app/src/main/java/app/revanced/manager/ui/component/AutoUpdatesDialog.kt

```diff
@@ -37,79 +32,49 @@ fun AutoUpdatesDialog(onSubmit: (Boolean, Boolean) -> Unit) {
             }
         },
-        icon = {
-            Icon(Icons.Outlined.Update, null)
-        },
-        title = {
-            Text(
-                text = stringResource(R.string.auto_updates_dialog_title),
-                style = MaterialTheme.typography.headlineSmall.copy(textAlign = TextAlign.Center),
-                color = MaterialTheme.colorScheme.onSurface,
-            )
-        },
+        icon = { Icon(Icons.Outlined.Update, null) },
+        title = { Text(text = stringResource(R.string.auto_updates_dialog_title)) },
         text = {
             Column(
-                horizontalAlignment = Alignment.CenterHorizontally,
-                verticalArrangement = Arrangement.spacedBy(16.dp),
+                verticalArrangement = Arrangement.spacedBy(8.dp),
             ) {
-                Text(
-                    text = stringResource(R.string.auto_updates_dialog_description),
-                    style = MaterialTheme.typography.bodyMedium,
-                )
+                Text(text = stringResource(R.string.auto_updates_dialog_description))
 
-                AutoUpdatesItem(
-                    headline = R.string.auto_updates_dialog_manager,
-                    icon = Icons.Outlined.Update,
-                    checked = managerEnabled,
-                    onCheckedChange = { managerEnabled = it }
-                )
-                HorizontalDivider()
-                AutoUpdatesItem(
-                    headline = R.string.auto_updates_dialog_patches,
-                    icon = Icons.Outlined.Source,
-                    checked = patchesEnabled,
-                    onCheckedChange = { patchesEnabled = it }
-                )
+                Column {
+                    AutoUpdatesItem(
+                        headline = stringResource(R.string.auto_updates_dialog_manager),
+                        icon = Icons.Outlined.Update,
+                        checked = managerEnabled,
+                        onCheckedChange = { managerEnabled = it }
+                    )
+                    HorizontalDivider()
+                    AutoUpdatesItem(
+                        headline = stringResource(R.string.auto_updates_dialog_patches),
+                        icon = Icons.Outlined.Source,
+                        checked = patchesEnabled,
+                        onCheckedChange = { patchesEnabled = it }
+                    )
+                }
 
-                Text(
-                    text = stringResource(R.string.auto_updates_dialog_note),
-                    style = MaterialTheme.typography.bodyMedium,
-                    color = MaterialTheme.colorScheme.onSurfaceVariant,
-                    modifier = Modifier.fillMaxWidth()
-                )
+                Text(text = stringResource(R.string.auto_updates_dialog_note))
             }
         }
     )
 }
 
 @Composable
 private fun AutoUpdatesItem(
-    @StringRes headline: Int,
+    headline: String,
```

Comment:
I thought it would be more consistent since official components do it like this too

Reply:
Some of our own components use string resource ids as parameters, so it would still be consistent with those. Making our code consistent with the official components is not wrong, but that is probably better left for another PR.